### ISSUE TITLE
Not passing the props down

### ISSUE THREAD

**Issue by idesignpixels** [jedwards1211/react-meteor-data#2]
I have been following several guides but just cannot get the props to go to the child component.
I get them in component will receive props as the argument but it doesn't update the props, I always get an empty array in the Component, the pubsub is working as i can log them out and meteortoys show the collection.

I first had this problem with orionsoft:react-meteor-data then backtracked to react-meteor-data with the same problem.

As the guides have been followed exactly could this be an issue with some of the new react deprecations?

```
export default CampaignListContainer = createContainer(props => {
  const handler = Meteor.subscribe("campaigns");
  const isLoading = !handler.ready();
  let campaignList = Campaigns.find().fetch();
  let hasCampaigns = !!campaignList;
  console.log(campaignList);
  return {campaigns: hasCampaigns ? campaignList : [], isLoading: isLoading};
}, CampaignList);
```

```
export default class CampaignList extends React.Component {
  constructor(props) {
    super(props);
  }

  componentWillReceiveProps(newProps) {
    console.log(newProps); // i get the right data!
    console.log(this.props); // empty array >_<
  }

  render() {
    return (
      <div className={styles.campaignList}>
        <h1>Campaigns</h1>
        {this.props.campaigns.map(campaign => {
          <div>a campaign</div>
        })}
      </div>
    );
  }
}

CampaignList.propTypes = {
  campaigns: PropTypes.array,
}

```

**Comment by jedwards1211** [comment]
@Lindstrom1989 I wonder if it could be an issue.  I copied this code straight out of the official meteor package...and I can't say I like the code, it's full of hacks.  To be honest I don't use this myself anymore.  We should probably look into writing a much cleaner replacement for this that works more like `react-redux`'s `connect`.

**Comment by idesignpixels** [comment]
Yeah i have been trying to use https://github.com/orionsoft/react-meteor-data to that effect but it seems to have a undeclared dependancy on this package? could you release the latest version on atmosphere as it seems to be 2 behind and someone has got create class in a separate import now in the most recent pr.

**Comment by idesignpixels** [comment]
What do you you now out of interest?, i tried cloning the latest version and still have the same problems, not sure whats going on.

**Comment by jedwards1211** [comment]
@Lindstrom1989 I have a bunch of code that stores documents from Mongo collections in my Redux state.  Yeah, when I find time I'll see if there have been any changes to the official `react-meteor-data` package and integrate them.

And `orionsoft/react-meteor-data` doesn't depend on this package; it depends on the official `react-meteor-data` *Meteor package*.  This is just an unofficial *npm package* version of it that I created.  Sorry for the confusion.
